### PR TITLE
Tag OnlineStats.jl v0.9.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [joshday/OnlineStats.jl](https://github.com/joshday/OnlineStats.jl)
Release: [v0.9.0](https://github.com/joshday/OnlineStats.jl/releases/tag/v0.9.0)
Travis: [Travis Build Status](https://travis-ci.org/joshday/OnlineStats.jl/builds/248373271)
Diff: [vs v0.8.0](https://github.com/joshday/OnlineStats.jl/compare/7f31ade7d0c555ecd201438592510ba289154faf...931c9d4c3ee899131d4da6b22398d5f43da9255b)
`requires` vs v0.8.0: 
```diff
--- 0.8.0/requires
+++ 0.9.0/requires
@@ -1,4 +1,9 @@
-julia 0.6.0-pre
+julia 0.6.0
+OnlineStatsBase
 StatsBase
 LearnBase
 Distributions
+SweepOperator
+LossFunctions
+PenaltyFunctions
+RecipesBase
```
cc: @joshday

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

### CODE REVIEW

**Comment by tkelman** [comment]
It's generally not a good idea to extend Base functions, or functions from a different package, on types from a different package (https://docs.julialang.org/en/latest/manual/style-guide.html#Avoid-type-piracy-1). That can change the behavior of unrelated code, and obscure where method dependencies are actually coming from.

**Comment by tkelman** [comment]
Do you plan on refactoring this on top of https://github.com/JuliaLang/METADATA.jl/pull/10056 or doing that for a future tag?

**Comment by joshday** [comment]
Yes, OnlineStats has been updated accordingly!  I'll make a new release when I'm back at a computer.

File: OnlineStats/versions/0.9.0/requires [hunk 1]
```diff
@@ -0,0 +1,9 @@
+julia 0.6.0
+OnlineStatsBase
```
Comment (tkelman) [inline OnlineStats/versions/0.9.0/requires]: this wouldn't work with any versions earlier than the most recent tag, would it?

Comment (joshday) [inline OnlineStats/versions/0.9.0/requires]: Correct...new tag coming in a moment.

**Pull request merged**